Dockerfile

```diff
@@ -58,4 +60,44 @@ RUN cp ./target/sparkml-serving-3.3.jar /usr/local/lib/sparkml-serving-3.3.jar
 RUN cp ./serve.sh /usr/local/bin/serve.sh
 
 RUN chmod a+x /usr/local/bin/serve.sh
-ENTRYPOINT ["/usr/local/bin/serve.sh"]
+
+# remove the maven-shared-utils packages - older versions create vulnerabilities
+RUN find / -depth -name maven-shared-utils -type d -exec rm -r "{}" \;
+
+# remove the surefire packages - contains versions of maven-shared-utils that create vulnerabilities
+RUN find / -depth -name surefire -type d -exec rm -r "{}" \;
```

Comment:
Can all of these be combined into the same `RUN` command?

Reply:
Yes. They can be combined.